### PR TITLE
Add event status to event table

### PR DESCRIPTION
**Pull request by tsteward**
This adds the status for each event to the event table to give an overview of how all of the events are progressing in one place.
screencapture-localhost-8088-1489855741573

### CODE REVIEW

**Comment by tsteward** [comment]
The `event_table` template is also used in the event list page. I don't think we want also do this on that page because it is cached for longer cache expiration (I think). Would there be a way to make the `event_table` template only show details on main page.

File: templates/event_partials/event_table.html [hunk 1]
```diff
@@ -14,6 +14,14 @@
           {% if event.city_state_country %}
             <br>
             <small>{{event.city_state_country}}</small>
+            <br>
+            {% if event.now and event.next_match %}
+              <b>Next Match: </b> {{event.next_match.verbose_name}}
+            {% else %}
+              {% if not event.alliance_selections and event.previous_match %}
```
Comment (fangeugene) [inline templates/event_partials/event_table.html]: The entire logic should be wrapped in event.now, not just the first clause.

**Comment by fangeugene** [comment]
The cache is short on the event list.

File: templates/event_partials/event_table.html [hunk 2]
```diff
@@ -14,6 +14,16 @@
           {% if event.city_state_country %}
             <br>
             <small>{{event.city_state_country}}</small>
+            <br>
```
Comment (fangeugene) [inline templates/event_partials/event_table.html]: This `<br>` should be wrapped inside the if statements below, before each `<b>`

Comment (tsteward) [inline templates/event_partials/event_table.html]: Yep just noticed

**Comment by fangeugene** [comment]
This looks good. Do you have any other changes you'd like to make?

**Comment by tsteward** [comment]
No

**Comment by jaredhasenklein** [comment]
I just looked at the live home page for the first time since this change was made. I like that the information is made available however, I think that the formatting needs some work.

The event name and next match are the same font size whereas the location is 1.5 points smaller. This 
"Next Match" text is bold and given that it is the same size as the event name, I think it either needs to be unbolded and the event should be bolded OR the entire Next Match line should be reduced to the same size as the location.

Thoughts?

**Comment by jaredhasenklein** [comment]
screenshot 315

@tsteward @fangeugene

**Comment by fangeugene** [comment]
I wouldn't mind small and no bold.

**Comment by tsteward** [comment]
@jaredhasenklein Do you want to go ahead and make the change?

**Comment by jaredhasenklein** [comment]
Doing it now

**Comment by jaredhasenklein** [comment]
Proposed in #1846

**Comment by jaredhasenklein** [comment]
should be closed

**Pull request merged**